ui/src/app/pools/components/TagHeaderForms/TagHeaderForms.test.tsx

```diff
@@ -1,17 +0,0 @@
-import { render, screen } from "@testing-library/react";
```

Comment:
This was moved to [ui/src/app/tags/components/TagsHeader/TagHeaderForms/TagHeaderForms.test.tsx](https://github.com/canonical-web-and-design/maas-ui/pull/3672/files#diff-80902f3be3a0ae2c0cbef9941edbdc7d858007e9a5733db7c354e190153a7dd0)